templates/README.mustache

```diff
@@ -72,9 +72,8 @@ To install the bindings via [Composer](https://getcomposer.org/), add the follow
 ```json
 {
     "require": {
-        "{{packageName}}": "dev-openapi"
-    },
-    "minimum-stability": "dev"
+        "{{packageName}}": "{{artifactVersion}}"
```

Comment:
Apologies about my previous comment, but let's actually use `packageVersion`, not `artifactVersion` both here and on line 85 (`composer require {{packageName}}:{{artifactVersion}}` -> `composer require {{packageName}}:{{packageVersion}}`)

This is because the caret `^` [allows grabbing the latest in a specific range](https://getcomposer.org/doc/articles/versions.md#caret-version-range-).

With `packageVersion` the generated code will look like this:

```
"hellosign/hellosign-php-sdk": "^6.0.0"
```

and users will be able to install it manually using this:

```
composer require hellosign/hellosign-php-sdk:^6.0.0
```